e2e/types.go

```diff
@@ -197,3 +197,11 @@ func (s *Scenario) PrepareVMSSModel(ctx context.Context, t *testing.T, vmss *arm
 		vmss.Tags[buildIDTagKey] = &config.Config.BuildID
 	}
 }
+
+func (s *Scenario) IsWindows() bool {
+	return s.VHD.OS == config.OSWindows
```

Comment:
Could also put this method on the VHD class (type *Image) - isOsWindows or something like that. Sadly the OS is just a string otherwise that would be the obvious place to put it.

Nice having it on the Scenario class too tho.

Reply:
We do have a method "IsWindowsSIGDistro" on the Distro class, but not sure if that's accessible from here.

Reply:
I'm not sure where is the best place to put it.

It seems like a comman check we perform. And we pass `*Scenario` around.
Given the frequence of the usage I put it the easit to access spot.